### PR TITLE
Add more docs on installation, running, and configuration.

### LINKED ISSUE #16: Installation / Getting Started Instructions

### LINKED ISSUE #51: add command line instructions to docs
I think once we do that we can start promoting the package?

### PR DESCRIPTION
**Pull request by ian-r-rose**
Fixes #16 , fixes #51

### CODE REVIEW

**Comment by ian-r-rose** [comment]
A few things I noticed:

1. In the AONI dialog it is still called "Area of Interest"
1. There are some AOI related options in the left pane (opacity, etc). Should we add corresponding options for AONI?

**Comment by vahedi25411** [comment]
> A few things I noticed:
> 
> 1. In the AONI dialog it is still called "Area of Interest"
> 2. There are some AOI related options in the left pane (opacity, etc). Should we add corresponding options for AONI?

Thank you for letting me know. Sure, I will fix these issues and make another pull request for the area of non-interest.

**Comment by ian-r-rose** [comment]
@vahedi25411 oops, looks like I commented on the wrong issue, I meant to do that on the AONI PR (#83). In any event, can you take a look at the doc changes here to see if they look okay (adding a dockerfile notwithstanding)?

File: README.md [hunk 1]
```diff
@@ -2,61 +2,137 @@
...
+
+You can run it via the command line by using the `cli` argument:
+```bash
+automated-walk-bike-counter --cli True
+```
+Since this way of running the application doesn't have dialogs for selecting videos
+and other options, you will need to provide more command line arguments.
 
-## Configuration
+For instance, to run the algorithm on a local video file with CSV exports at one minute intervals,
```
Comment (vahedi25411) [inline README.md:109]: I think having CSV export here without explaining the CSV export feature can be confusing. 
The location of the CSV file should also be specified here.

Comment (ian-r-rose) [reply README.md:121]: Good point, I added a note about CSV and video exports, and where they are placed.

**Pull request merged**